tests/test_info.py

```diff
@@ -51,16 +51,23 @@ def test_info():
     assert_in(conda_info_s_out, conda_info_all_out)
 
 
+@pytest.mark.skipif(
+    context.subdir not in ("linux-64", "osx-64", "win-32", "win-64", "linux-32"),
+    reason="Skip unsupported platforms",
+)
 @pytest.mark.integration
 def test_info_package_json():
-    out, err, rc = run_command(Commands.INFO, "--json", "itsdangerous=1.0.0=py37_0")
+    # This is deprecated
```

Comment:
What do you mean? Does that mean it should not be tests? Please elaborate in the comment